packages/components/src/box-control/input-control.tsx

```diff
@@ -179,7 +179,7 @@ export default function BoxInputControl( {
 		? getPresetIndexFromValue( mergedValue, presetKey, presets )
 		: undefined;
 	const marks = hasPresets
-		? [ { value: 0, label: '', tooltip: __( 'None' ) } ].concat(
+		? [ { value: 0, label: '', tooltip: __( 'None' ) as string } ].concat(
```

Comment:
Why did a type assertion `as string` become necessary here? Does that have something to do with the new `TranslatableText` type?

Reply:
Yes, because this is an array with the item `{ value: number, label: string, tooltip: TranslatableText }`, trying to concat with items of type `{ value: number, label: string, tooltip: string }`